mmv1/products/vertexai/FeatureGroupFeature.yaml

```diff
@@ -0,0 +1,109 @@
+# Copyright 2023 Google Inc.
```

Comment:
nit: 2024 (apologies 🙂 )

Reply:
I needed this change 😀 Do we want to have a separate PR? https://github.com/GoogleCloudPlatform/magic-modules/pull/9692/commits/6afed03f0a04967a7a5e0a17c222fa0c4ac0f6f4